go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/cli/safeexec v1.0.1
 	github.com/cpuguy83/go-md2man/v2 v2.0.5
 	github.com/creack/pty v1.1.24
+	github.com/digitorus/timestamp v0.0.0-20231217203849-220c5c2851b7
```

Comment:
I really wish this module had releases.  That said, I see @malancas and @bdehamer have contributed before and this is just explicitly depending upon this previously indirect dependency.